cnf-tests/testsuites/e2esuite/dpdk/dpdk.go

```diff
@@ -1069,3 +1252,38 @@ func getDeviceRXBytes(pod *corev1.Pod, device string) (int, error) {
 	}
 	return -1, fmt.Errorf("could not find RX stats: %v", statsLines)
 }
+
+func isContainerUseDevicesSEBooleanDisabled(node string) bool {
+	mcd := getMachineConfigDaemon(node)
+
+	output, err := pods.ExecCommand(client.Client, *mcd, []string{"sh", "-c", "nsenter --mount=/proc/1/ns/mnt -- sh -c 'getsebool container_use_devices'"})
+	Expect(err).ToNot(HaveOccurred())
+
+	return strings.Contains(output.String(), "off")
+}
+func setContainerUseDevicesSEBoolean(node string) {
+	mcd := getMachineConfigDaemon(node)
+
+	_, err := pods.ExecCommand(client.Client, *mcd, []string{"sh", "-c", "nsenter --mount=/proc/1/ns/mnt -- sh -c 'setsebool container_use_devices 1'"})
+	Expect(err).ToNot(HaveOccurred())
+}
+
+func unsetContainerUseDevicesSEBoolean(node string) {
+	mcd := getMachineConfigDaemon(node)
+
+	_, err := pods.ExecCommand(client.Client, *mcd, []string{"sh", "-c", "nsenter --mount=/proc/1/ns/mnt -- sh -c 'setsebool container_use_devices 0'"})
+	Expect(err).ToNot(HaveOccurred())
+}
+
+func getMachineConfigDaemon(node string) *corev1.Pod {
+	listOptions := metav1.ListOptions{
+		FieldSelector: fields.SelectorFromSet(fields.Set{"spec.nodeName": node}).String(),
+		LabelSelector: labels.SelectorFromSet(labels.Set{"k8s-app": "machine-config-daemon"}).String(),
+	}
+
+	mcds, err := sriovclient.Pods(utils.NamespaceMachineConfigOperator).List(context.Background(), listOptions)
+	Expect(err).ToNot(HaveOccurred())
+	Expect(mcds.Items).ToNot(BeEmpty(), "Cluster machines are not managed by the machine operator")
+
+	return &mcds.Items[0]
+}
```

Comment:
can you please move this functions out if the DPDK test to the utils please

Reply:
sure

Reply:
done